instagram/src/components/PostContainer/PostsPage.js

```diff
@@ -0,0 +1,50 @@
+import React, { Component } from 'react';
+import SearchBar from '../SearchBar/SearchBar';
+import PostContainer from './PostContainer';
+import dummyData from '../../dummy-data';
+
+class PostsPage extends Component {
+  constructor() {
+    super();
+    this.state = {
+      posts: [],
+      search: []
+    };
+  }
+
+  componentDidMount() {
+    this.setState({ posts: dummyData });
+  }
+
+  search = event => {
+    const newPosts = this.state.posts.filter(post => {
+      if (post.username.includes(event.target.value)) {
```

Comment:
I believe this can be reduced to `return post.username.includes(event.target.value)` since filter just needs to return true or false to decide whether or not to return post and `.includes` by default returns a boolean.